std/algorithm/mutation.d

```diff
@@ -116,9 +116,6 @@ Params:
 
 Returns:
     The number of elements brought to the front, i.e., the length of `back`.
-
-See_Also:
-    $(HTTP sgi.com/tech/stl/_rotate.html, STL's rotate)
```

Comment:
I would keep this one for search-ability since the names are different.

Reply:
Ok. Updated.